docs/pages/Getting_Started.md

```diff
@@ -2243,13 +2263,14 @@ and a capability to do a standard input driven interactive definition of the
 observation definition.
 
 DART makes a further distinction between specific *observation types* and
-generic *quantities*. The role of the various obs_def input files
+generic *quantities*. Essentially, an *observation type* can be considered to be instrument-specific and the *quantity* is something you learned about in physics.
+As an example, 'RADIOSONDE_TEMPERATURE' is an *observation type* of the fundamental *quantity* 'TEMPERATURE'. The role of the various obs_def input files
 is to define the mapping between the types and quantities, and optionally to
-provide type-specific processing routines.
+provide type-specific processing routines. Specifying many observation types allows DART to be able to evaluate some observations and assimilate others - even if both instrument measure the same *quantity*.
```

Comment:
"both instrument" should be changed here. Also the beginning of the sentence says "many" but the end says "both." How about 

Specifying many observation types allows DART to be able to evaluate some observations and assimilate others - even if many instruments measure the same *quantity*.

Reply:
Thanks ... I opted for.  '... - even if the instruments measure the same _quantity_.'